app/lib/lti_advantage/params.rb

```diff
@@ -0,0 +1,58 @@
+module LtiAdvantage
+  # This is for extracting data from the lti jwt in more human-readable ways
+  class Params
+    attr_reader :token
+
+    def initialize(lti_token)
+      @token = lti_token
+    end
+
+    def launch_context
+      # This is an array, I'm not sure what it means to have more than one
```

Comment:
Here's a link to the context claim in the specification - https://www.imsglobal.org/spec/lti/v1p3/#context-claim-0

It says the array must contain at least one type from this list:
https://www.imsglobal.org/spec/lti/v1p3/#context-type-vocabulary

You can have multiple contexts. To be compatible with other LMSs we might want to use an include? rather than [0] but for now this should be fine.